CMakeLists.txt

```diff
@@ -75,7 +75,6 @@ else()
         set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-psabi")
         set(CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,--no-undefined")
     elseif(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-        set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-deprecated-builtins")
```

Comment:
The flag is available from clang 15 onwards.
It would be better to add a check on `CMAKE_CXX_COMPILER_VERSION` instead of completely removing the flag.

Reply:
Good call.  I've done that now in 17a0a6fef29e54bb5f6635b5ade52c08c743123a